src/components/common/AddEditOrgPerson.vue

```diff
@@ -54,11 +54,12 @@
             </p>
           </v-card>
 
+          <div v-if="isCompletingParty && !isRoleStaff && isTypeCoop" class="mt-8" />
```

Comment:
I prefer this explicit white-space block (which is needed only when the block above is rendered), rather than an un-obvious conditional class on the form below. (It took me a while to understand why the conditional class existed.)